Makefile

```diff
@@ -65,11 +70,15 @@ soci: FORCE
 check:
 	cd scripts/ ; ./check-all.sh
 
-flatc:
-	rm -rf $(CURDIR)/ztoc/fbs/ztoc
-	flatc -o $(CURDIR)/ztoc/fbs -g $(FBS_FILE_PATH)
-	rm -rf $(CURDIR)/ztoc/compression/fbs/zinfo
-	flatc -o $(CURDIR)/ztoc/compression/fbs -g $(FBS_FILE_PATH_COMPRESSION)
+flatc: $(ZTOC_FBS_GO_FILES) $(COMPRESSION_FBS_GO_FILES)
```

Comment:
nit: this should be marked phony since `flatc` isn't a file that gets generated by this rule.